src/primitives/block.cpp

```diff
@@ -259,6 +275,31 @@ bool CBlock::CheckBlockSignature() const
 
         return pubkey.Verify(GetHash(), vchBlockSig);
 
+    }
```

Comment:
indentation

Reply:
I can't see the indention wrong. After I rewrote the indention to space (previously it was space too), git doesn't show any diff.
I removed the blank line at line 277.